shark/shark_benchmark_runner.py

```diff
@@ -76,7 +76,7 @@ def benchmark_torch(self, modelname):
         torch_device = torch.device(
             "cuda:0" if self.device == "gpu" else "cpu"
         )
-        HFmodel, input, act_out = get_torch_model(modelname)
+        HFmodel, input, _ = get_torch_model(modelname)
```

Comment:
If you know what this is, even if it doesn't get used, its better to give it a name, or not assign it and append "[:2]" to get_torch_model(modelname) (assuming it returns a list or tuple)